src/main/java/com/uid2/admin/vertx/service/EncryptionKeyService.java

```diff
@@ -157,11 +170,14 @@ private void handleKeyList(RoutingContext rc) {
 
     private void handleRotateMasterKey(RoutingContext rc) {
```

Comment:
This will now return the keys in a different order - it will return all the MasterKeySite first, and then all the RefreshKeySite second (in the same json file). Does this matter? I don't know who calls it, so not sure

Reply:
i dont think it matters as it's just a unsorted list in json. @aulme any view?

Reply:
I guess it's fine to not preserve order if we don't care about (which I think we don't). Lists in JSON are ordered however.